gui/default/index.html

```diff
@@ -788,6 +788,12 @@ <h4 class="panel-title">
         </div>
         <div class="form-group">
           <span class="pull-right">
+            <button type="button" class="btn btn-sm btn-default" ng-click="setAllDevicesPause(true)" ng-if="isAtleastOneDevicePausedStateSetTo(false)">
+             <span class="fas fa-pause"></span>&nbsp;<span translate>Pause All</span>
+            </button>
+            <button type="button" class="btn btn-sm btn-default" ng-click="setAllDevicesPause(false)" ng-if="isAtleastOneDevicePausedStateSetTo(true)">
+             <span class="fas fa-play"></span>&nbsp;<span translate>Resume All</span>
```

Comment:
minor nit: please use the same indentation of two spaces.

Reply:
Sorry. I did't notice it. It is fixed now.